OpenRA.Mods.Common/Traits/World/EditorActorPreview.cs

```diff
@@ -98,7 +107,9 @@ public void Tick()
 
 		public IEnumerable<IRenderable> Render()
 		{
-			return previews.SelectMany(p => p.Render(worldRenderer, CenterPosition));
+			var items = previews.SelectMany(p => p.Render(worldRenderer, CenterPosition))
+				.Select(y => !ShowBorder ? y : DateTime.Now.Millisecond < 500 ? y : y.WithPalette(worldRenderer.Palette("highlight")));
+			return !ShowBorder ? items : items.Concat(new IRenderable[] { SelectionBox });
```

Comment:
In order to keep the visuals consistent with the rest of the game, would you mind changing this to:
```csharp
public IEnumerable<IRenderable> Render()
{
	var items = previews.SelectMany(p => p.Render(worldRenderer, CenterPosition));
	if (ShowBorder)
	{
		var highlight = worldRenderer.Palette("highlight");
		var overlay = items.Where(r => !r.IsDecoration)
			.Select(r => r.WithPalette(highlight));
		return items.Concat(overlay).Append(SelectionBox);
	}

	return items;
}
```
and keeping the selection box width as 1?